src/controller/cve.controller/index.js

```diff
@@ -81,25 +68,22 @@ router.get('/cve',
   /*
   #swagger.tags = ['CVE Record']
   #swagger.operationId = 'cveGetFiltered'
-  #swagger.summary = "Retrieves all CVE records (accessible to all users)"
+  #swagger.summary = "Retrieves all CVE entries (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
         <p>Endpoint is accessible to all</p>
         <h2>Expected Behavior</h2>
-        <p><b>All users:</b> Retrieves all CVE records for all organizations</p>"
+        <p><b>All users:</b> Retrieves all CVE entries for all organizations</p>"
   #swagger.parameters['$ref'] = [
     '#/components/parameters/cveRecordFilteredTimeModifiedLt',
     '#/components/parameters/cveRecordFilteredTimeModifiedGt',
     '#/components/parameters/cveState',
     '#/components/parameters/countOnly',
     '#/components/parameters/assignerShortName',
```

Comment:
descriptions for assignerShortName and assigner should say 
Filter by assignerShortName
Filter by assigner org UUID